lib/spack/docs/basic_usage.rst

```diff
@@ -1,4 +1,4 @@
-.. Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+.. Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
Can you revert this change?

Reply:
Done.
I think I made a bad git command. Sorry

Reply:
No worries, and thanks for your contributions!